src/lua/init.c

```diff
@@ -592,7 +594,11 @@ static int
 tarantool_panic_handler(lua_State *L) {
 	const char *problem = lua_tostring(L, -1);
 #ifdef ENABLE_BACKTRACE
-	print_backtrace();
+	struct backtrace bt;
+	backtrace_collect_frames(&bt, fiber(), 2);
+	char *bt_buf = tt_static_buf();
```

Comment:
`tt_static_buf` is fairly small (1024 bytes) - the backtrace may not fit. The old version used `SMALL_STATIC_SIZE` (12 Kbytes), which should be enough. However, IMO ideally the function should print to stderr directly, without a temporary buffer.

Reply:
Fixed: added `backtrace_print` for this case.